doc/nrf/libraries/caf/net_state.rst

```diff
@@ -13,14 +13,14 @@ The module provides different backends for available networks (like LTE or OpenT
 Configuration
 *************
 
-You can enable |net_state| by selecting the :kconfig:option:`CONFIG_CAF_NET_STATE` option in the configuration.
+To enable the |net_state|, select the :kconfig:option:`CONFIG_CAF_NET_STATE` Kconfig option in the configuration.
```

Comment:
```suggestion
To enable the |net_state|, set the :kconfig:option:`CONFIG_CAF_NET_STATE` Kconfig option in the configuration.
```
`select` is a Kconfig term, I think using `set` may be better because of that